topic_monitor/scripts/data_publisher.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2016 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import sys
+from time import sleep
+
+import rclpy
+from rclpy.qos import QoSDurabilityPolicy, QoSHistoryPolicy, QoSReliabilityPolicy
+from rclpy.qos import QoSProfile
+
+from std_msgs.msg import Header
+
+
+default_depth = 10
+
+
+def main():
+    parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument(
+        'data_name',
+        nargs='?',
+        default='topic1',
+        help='Name of the data (must comply with ROS topic rules)')
+
+    parser.add_argument(
+        '--best-effort',
+        action='store_true',
+        default=False,
+        help='Set QoS reliability option to "best effort"')
+
+    parser.add_argument(
+        '--transient-local',
+        action='store_true',
+        default=False,
+        help='Set QoS durability option to "transient local"')
+
+    parser.add_argument(
+        '--depth',
+        type=int,
+        default=default_depth,
+        action='store',
```

Comment:
Isn't store the default behavior of argparse? if so all `action='store'` can be removed, in all files